doc/httpcli.md

```diff
@@ -0,0 +1,17 @@
+# HTTP endpoint
+
+Livepeer node exposes REST endpoint which is used by `livepeer_cli` for communication with node.
```

Comment:
"The Livepeer node exposes a HTTP interface for monitoring and managing the node. This is how the `livepeer_cli` tool interfaces with a running node."

Reply:
fixed